src/charm.py

```diff
@@ -109,6 +113,36 @@ def __init__(self, *args):
             self._certificates.on.certificate_expiring, self._on_certificate_expiring
         )
 
+    def _on_install(self, event: InstallEvent) -> None:
+        client = Client()
+        logger.info("Creating external AMF service")
+        client.create(
```

Comment:
Why are we creating an additional service instead of leveraging the one we already have? There can be multiple port objects in 1 service.